static-web/index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>HTML and CSS Quiz</title>
+	<link rel="stylesheet" href="/css/quiz.css">
+</head>
+<body>
+  <div class="allCards clearfix">
+    <!-- Begin Card -->
+  	<article class="card">
+  		<header class="cardTitle">
```

Comment:
For some reason this is not what it looks like for me. Do you know why it would look different on my sublime and here when everything is up to date? Poor indentation bothers me so it's driving me mad that I can't figure out why I am seeing something totally different.